File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/DataInputHistoryResponse.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.DataInputHistoryResponsePoints
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param id The ID of this data input
 * @param name Name of this data input
 * @param points Data points from this data input
 */
data class DataInputHistoryResponse (

        @get:NotNull 
        @JsonProperty("name") val name: String,

        @JsonProperty("id") val id: Long? = null,

        @JsonProperty("points") val points: List<DataInputHistoryResponsePoints>? = null
) {

}

File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/DataInputHistoryResponse.scala

package org.openapitools.client.model


case class DataInputHistoryResponse (
    /* The ID of this data input */
    _id: Option[Long],
    /* Name of this data input */
    _name: String,
    /* Data points from this data input */
    _points: Option[List[DataInputHistoryResponsePoints]]
)
object DataInputHistoryResponse {
    def toStringBody(var_id: Object, var_name: Object, var_points: Object) =
        s"""
        | {
        | "id":$var_id,"name":$var_name,"points":$var_points
        | }
        """.stripMargin
}
